db/machine.go

```diff
@@ -82,11 +323,11 @@ func (d DB) GetMachines() ([]Machine, error) {
 
 	machines := []Machine{}
 	for rows.Next() {
+
 		m := Machine{db: &d}
 		err = rows.Scan(
-			&m.id, &m.Name, &m.Architecture, &m.Microarchitecture,
```

Comment:
A pull request like this that adds functionality should be done independently of formatting-only changes such as this hunk.